daemon/algod/api/server/v2/account.go

```diff
@@ -422,6 +424,23 @@ func AppParamsToApplication(creator string, appIdx basics.AppIndex, appParams *b
 	return app
 }
 
+// sanitizePrintableUTF8String checks to see if the entire string is a UTF8 printable string.
+// If this is the case, the string is returned as is. Otherwise, the empty string is returned.
+func sanitizePrintableUTF8String(in string) string {
```

Comment:
What is the reason for not sharing this with daemon/algod/api/server/v1/handlers/handlers.go ?

Reply:
v1 is going to be deprecated, and when that happen, the code-duplication would no longer be an issue. But I agree that in general we should avoid code duplication.

Reply:
You can put this in `v2/utils.go`.